api/routes/map.js

```diff
@@ -109,7 +109,8 @@ async function router(schema, config) {
 
         res.writeHead(200, {
             'Content-Type': 'application/vnd.mapbox-vector-tile',
-            'Content-Encoding': 'gzip'
+            'Content-Encoding': 'gzip',
```

Comment:
Make sure you only send these compressed if the client asks for it. The client will send a header like `accept-encoding: gzip, deflate, br` to tell you it can handle gzipped data.